esp_jpeg/Kconfig

```diff
@@ -66,4 +66,15 @@ menu "JPEG Decoder"
         config JD_FASTDECODE_TABLE
             bool "+ Table conversion for huffman decoding (wants 6 << HUFF_BIT bytes of RAM)"
         endchoice
+
+    config JD_DEFAULT_HUFFMAN
```

Comment:
Should we add a line to README.md to mention this new option? It' not easily discoverable since CONFIG_JD_USE_ROM is enabled by default, so CONFIG_JD_DEFAULT_HUFFMAN is not visible in the menu.